### PR TITLE
fix(service-portal): audit + minor fixes for Access control

### PR DESCRIPTION
**Pull request by disaerna**
# Access control - fixes

## What

- Add more detailed audit logs
- Revert button size
- Fix error check

## Why

Needed to remove feature flag ! 

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by thordurhhh — APPROVED** [review]
finance resolver LGTM 👍

File: libs/api/domains/auth/src/lib/resolvers/delegation.resolver.ts [hunk 1]
```diff
@@ -74,6 +76,13 @@ export class DelegationResolver {
       })
     }
 
+    this.auditService.audit({
+      auth: user,
+      namespace: '@island.is/api/auth',
+      action: 'createDelegation',
+      resources: [input.toNationalId],
+    })
+
```
Comment (saevarma) [inline libs/api/domains/auth/src/lib/resolvers/delegation.resolver.ts:85]: This is audit logged in the service so we don't need it in the resolver.

File: libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts [hunk 2]
```diff
@@ -108,22 +108,37 @@ export class FinanceResolver {
...
       user,
     )
+
+    this.auditService.audit({
+      auth: user,
+      namespace: '@island.is/api/finance',
+      action: 'getDocumentList',
+      resources: [input.listPath, input.dayFrom, input.dayTo],
+    })
+
```
Comment (saevarma) [inline libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts:118]: What is the reason to move from the decorator?
The decorator can be updated to log the resources, it logs the authentication be default and uses the namespace from the decorator on the controller.
I personally prefer the decorator over the service when I can.

Comment (eirikurn) [reply libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts:118]: Should add these as meta properties instead. The resources field is designed for specific resource ids which you want to query all audit entries for.

**Review by saevarma — APPROVED** [review]
LGTM

**Pull request merged**